zcash_client_backend/src/data_api/wallet.rs

```diff
@@ -316,7 +316,7 @@ pub fn spend<DbT, ParamsT, InputsT>(
     ovk_policy: OvkPolicy,
     min_confirmations: NonZeroU32,
 ) -> Result<
-    TxId,
+    Vec<TxId>,
```

Comment:
Update the documentation for this function to not assume a single transaction is created.